src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1111,7 +1111,14 @@ table,
 }
 
 .item-info .stab {
-	display: inline-block;
+	width: fit-content;
+	/* This min-height is needed to unify the height of the stab elements because some of them
+	   have emojis.
+	*/
```

Comment:
Idea:

```css
.stab::before {
  content: '\200b'; /* zero-width space */
  font-size: 1.25rem; /* emoji font size */
}
```

Maybe the `content` could be the actual emoji when applicable, determined by a CSS class?

Reply:
The `min-height` is mostly for items which don't have an emoji actually. Unless I misunderstood what you meant?

Reply:
Right. The zero-with space makes the element taller as if there were an emoji.

Reply:
Exactly.

Reply:
So you wouldn't need an arbitrary min-height.

Reply:
Sorry, I still don't get your point.

Reply:
Sorry, I guess this is all moot. I thought that the actual height of the emoji is not a predictable px value. I've had experiences with some fonts not having a predictable height.